app/goal_routes.py

```diff
@@ -0,0 +1,96 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.goal import Goal
+from app import db
+from .task_routes import validate_model
+from app.models.task import Task
+
+goal_bp = Blueprint("goals", __name__, url_prefix="/goals")
```

Comment:
This can just be called bp since you've only got one blueprint in this file and you have routes for each model in their own file.